TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -559,41 +533,40 @@ class RouteOptionsViewController: UIViewController {
         }
     }
 
-    private func processRequest(result: Result<Response<RouteSectionsObject>>, requestURL: String, endPlace: Place) {
-        JSONFileManager.shared.logURL(timestamp: Date(), urlName: "Route requestUrl", url: requestURL)
+    private func processRequest(start: Place, end: Place, time: Date, type: SearchType) {
+        if let result =  getRoutes(start: start, end: end, time: time, type: type) {
+            result.observe(with: { [weak self] result in
+                guard let self = self else { return }
+                DispatchQueue.main.async {
+                    switch result {
+                    case .value(let response):
 
-        switch result {
-        case .value(let response):
+                        // Parse sections of routes
+                        [response.data.fromStop, response.data.boardingSoon, response.data.walking]
+                            .forEach { (routeSection) in
```

Comment:
@kevinchan159 is it convention to have `(routeSection)` here or just `routeSection`?

Reply:
`routeSection`